### PR TITLE
Create app Roku Remote

### PR DESCRIPTION
**Pull request by jhaygood86**
Creates a new app Roku Remote.

I wrote it, so I'm the copyright owner.

### CODE REVIEW

File: com.reaktix.RokuRemote.json [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+            ]
+        },
+        {
+            "name" : "rokuremote",
+            "builddir" : true,
+            "buildsystem" : "meson",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/jhaygood86/RokuRemote.git"
```
Comment (bilelmoussaoui) [inline com.reaktix.RokuRemote.json:68]:
```suggestion
                    "url" : "https://github.com/jhaygood86/RokuRemote.git",
```

Comment (jhaygood86) [reply com.reaktix.RokuRemote.json]: Fixed

**Comment by nedrichards** [comment]
bot, build com.reaktix.RokuRemote

**Comment by jhaygood86** [comment]
Compile error fixed.

**Comment by hfiguiere** [comment]
bot, build com.reaktix.RokuRemote

File: com.reaktix.RokuRemote.json [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/jhaygood86/RokuRemote.git",
+                    "tag" : "v0.2",
+                    "commit" : "8d3b15f7d7afbcdfd75ae3bcf78fbc68314250d0"
+                }
+            ]
+        }
+    ],
+    "build-options" : {
```
Comment (hfiguiere) [inline com.reaktix.RokuRemote.json:75]: it's minor, but this section is not needed, so you can just remove it.

**Comment by hfiguiere** [comment]
appstream file fails to validate.

File: com.reaktix.RokuRemote.json [hunk 3]
```diff
@@ -0,0 +1,79 @@
+{
+    "app-id" : "com.reaktix.RokuRemote",
```
Comment (barthalion) [inline com.reaktix.RokuRemote.json:2]: Is it an existing domain?

**Pull request closed without merging**